.circleci/config.yml

```diff
@@ -45,6 +45,8 @@ jobs:
   test:
     executor: node
     resource_class: medium+
+    environment:
+      MAX_WORKERS: 3 # Number of CPUs in resource_class
```

Comment:
`medium+` provides 3 CPUs, using an ENV variable so the number of CPUs is defined close to the resource_class